template.yaml

```diff
@@ -1,52 +1,164 @@
-                - !Sub ${FrontEnd.Arn}/*
+      Name: !Sub ${AWS::StackName}-Website
+      Artifacts:
+        Type: NO_ARTIFACTS
       Environment:
-        Variables:
-          BUCKET_NAME: !Ref FrontEnd
-          BUCKET_ARN: !GetAtt FrontEnd.Arn
-          XDG_CONFIG_HOME: /tmp/.config
-    Metadata:
-      StackeryName: deployFrontEnd
-  deployFrontEndDeployTrigger:
-    Type: Custom::FunctionDeployTrigger
+        ComputeType: BUILD_GENERAL1_SMALL
+        Image: aws/codebuild/amazonlinux2-x86_64-standard:3.0
+        Type: LINUX_CONTAINER
+        EnvironmentVariables: []
+      ServiceRole: !GetAtt WebsiteRole.Arn
+      Source:
+        Type: NO_SOURCE
+        BuildSpec: !Sub
+          - |-
+            version: 0.2
+            phases:
+              install:
+                runtime-versions:
+                  nodejs: latest
+                  python: latest
+                  ruby: latest
+                commands:
+                  - |
+                    _SOURCE_LOCATION=${SourceLocation}
+                    if [ s3 != "${!_SOURCE_LOCATION%%:*}" ]; then
+                      git clone ${SourceLocation} repo
+                      cd repo
+                      git checkout ${SourceVersion}
+                    else
+                      aws s3 cp ${SourceLocation} repo.tgz
+                      tar --strip-components 1 -xvvzf repo.tgz
+                    fi
+                  - cd ${SourceDirectory}
+              pre_build:
+                commands:
+                  - |
+                    if [ ! -f yarn.lock -a -f package.json ]; then
+                      npm install --production
+                    elif [ -f yarn.lock -a -f package.json ]; then
+                      yarn install --production
+                    elif [ -f requirements.txt ]; then
+                      pip install -r requirements.txt
+                    elif [ -f Gemfile ]; then
+                      bundle install
+                    fi
+              build:
+                commands:
+                  - ${BuildCommand}
+              post_build:
+                commands:
+                  - |
+                    _SOURCE_LOCATION=${SourceLocation}
+                    if [ s3 != "${!_SOURCE_LOCATION%%:*}" ]; then
+                      cd "${!CODEBUILD_SRC_DIR}/repo"
+                    else
+                      cd ${!CODEBUILD_SRC_DIR}
+                    fi
+                  - aws s3 sync '${PublishDirectory}' 's3://${DestinationBucketName}' --acl public-read --cache-control 'max-age=0, must-revalidate, public' --no-progress --delete
+          - PublishDirectory: src/site/build
+            BuildCommand: npm run build
+            SourceDirectory: src/site
+            DestinationBucketName: !Ref FrontEnd
+      Tags:
+        - Key: Stackery Project Type
+          Value: Website Builder
+  WebsiteEvents:
+    Type: AWS::Events::Rule
+    DependsOn: Website
+    Properties:
+      Name: !Sub ${SourceVersion}-stackerywebsite
```

Comment:
Remove this line in keeping with the website resource updates :)

Reply:
Thanks!